### DISCUSSION TITLE
Github commands

### DISCUSSION THREAD

**Discussion by Hasinikulla** [community/community#145087] [New to GitHub]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

github commands

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Hasinikulla ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category and including relevant details in your post like what you are trying to do in GitHub. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by miketopeak** [comment]
- Initialize a Repository:
git init
- Clone a Repository:
git clone <repository-url>
- Check Status:
git status
- Add Changes:
git add <file-name> or git add . (for all changes)
- Commit Changes:
git commit -m "Your commit message"
- Push Changes:
git push origin <branch-name>
- Pull Changes:
git pull origin <branch-name>
- Create a Branch:
git branch <branch-name>
- Switch Branches:
git checkout <branch-name>
- Merge Branches:
git merge <branch-name>
- View Commit History:
git log
- Discard Local Changes:
git checkout -- <file-name>

Hope that it works well..

**Reply by GabJuarez** [reply]
![image](https://github.com/user-attachments/assets/a304e49c-b861-47e6-b813-2a8334a54d6f)
![image](https://github.com/user-attachments/assets/9231c6fd-96c6-43e7-836b-8d48c6082e92)
Good luck! 😸

**Comment by ghostinhershell** [comment]
Hi @Hasinikulla! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
